test-case/verify-kernel-boot-log.sh

```diff
@@ -23,4 +23,21 @@ disable_kernel_check_point
 
 print_module_params
 
+# Check this device time is NTP Synchronized
+if check_ntp_sync; then
+    printf '\nTime Check: NTP Synchronized\n'
+else
+    timedatectl show
+    # try to disable/enable NTP once, this will trigger ntp sync twice,
+    # before stopping ntp and after enabling ntp.
+    re_enable_ntp_sync
+
+    if check_ntp_sync; then
```

Comment:
I feel like there should be a delay right after re-enabling but in the worst case only that particular test will fail so this can always be added later based on experience.